bundles/org.openhab.binding.keba/src/main/java/org/openhab/binding/keba/internal/handler/KeContactHandler.java

```diff
@@ -325,11 +325,25 @@ protected void onData(ByteBuffer byteBuffer) {
                         int state = entry.getValue().getAsInt();
                         switch (state) {
                             case 1: {
-                                updateState(CHANNEL_ENABLED, OnOffType.ON);
+                                updateState(CHANNEL_ENABLED_SYSTEM, OnOffType.ON);
                                 break;
                             }
                             default: {
-                                updateState(CHANNEL_ENABLED, OnOffType.OFF);
+                                updateState(CHANNEL_ENABLED_SYSTEM, OnOffType.OFF);
+                                break;
+                            }
+                        }
+                        break;
+                    }
+                    case "Enable user": {
+                        int state = entry.getValue().getAsInt();
+                        switch (state) {
+                            case 1: {
+                                updateState(CHANNEL_ENABLED_USER, OnOffType.ON);
+                                break;
+                            }
+                            default: {
+                                updateState(CHANNEL_ENABLED_USER, OnOffType.OFF);
                                 break;
                             }
                         }
```

Comment:
For your information, you could replace all this block by just:
```
updateState(CHANNEL_ENABLED_USER, OnOffType.from(entry.getValue().getAsInt() == 1));
```

Same for block just before.

Reply:
Good catch, that was copy & paste from the original code. I like the simplification and updated the PR.